### PR TITLE
Exports Classification

### PR DESCRIPTION
**Pull request by abhisheks008**
## Related Issue #222 
The goal of this project is to make a classification model which will identify the Exports of Canada with respect to the years.

- [x] LGM-SOC'21 Participant 
- [x] Contributor

Closes: #222 

## Description
The project is built on several Regression algortihms, such as, linear, lasso, ridge, decision tree, random forest, gradient boosting, Xgboost, support vector regressor and MLP regressor. Among these, the regression model with higher accuracy is to be chosen as the final model.

## Type of change

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
The models are being tested by the accuracy scores.

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by prathimacode-hub — APPROVED** [review]
Marvellous work. Regression lines on training set are shown fab. Simply perfect. 🤩✔@abhisheks008
